src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import './UserInfo.scss';
+
+export const UserInfo = ({
+  user,
+}) => (
```

Comment:
```suggestion
export const UserInfo = ({ user }) => (
```
You can make it in one line, as there is only 1 property